server/src/main/java/org/apache/iotdb/db/metadata/mtree/IMTreeBelowSG.java

```diff
@@ -136,8 +137,8 @@ List<MeasurementPath> getMeasurementPaths(PartialPath pathPattern, boolean isPre
    *
    * @param pathPattern a path pattern or a full path, may contain wildcard
    * @param isPrefixMatch if true, the path pattern is used to match prefix path
-   * @return Pair.left contains all the satisfied paths Pair.right means the current offset or zero
-   *     if we don't set offset.
+   * @return Pair.left contains all the satisfied getChildNodePathInNextLevel Pair.right means the
+   *     current offset or zero if we don't set offset.
    */
```

Comment:
Changes on this comment seems not appropriate.

Reply:
reverted it.